### PR TITLE
improve styling on videos

### LINKED ISSUE #130: Support Videos
- buttons on carousel
- width and height not correct

### PR DESCRIPTION
**Pull request by claireolmstead**
# Purpose
The goal of this PR is to improve the styling on videos + the carousel.

Closes #130 

## Solution
- Fix styling bugs
@wilwade 

### Change summary
* updated dot logic and styling
* updated image styling

## Steps to Verify
1. Post different dimension videos and make sure they look good.

### CODE REVIEW

File: backend/services/BroadcastDataBuilder.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export class BroadcastDataBuilder {
   }
 
   private static createAsset(asset: string) {
+    // TODO: not hard code the type as image.
```
Comment (claireolmstead) [inline backend/services/BroadcastDataBuilder.ts:13]:
```suggestion
```

File: frontend/src/features/BroadcastMedia/BroadcastMedia.module.css [hunk 2]
```diff
@@ -6,13 +6,18 @@
   overflow: hidden;
   max-height: 550px;
   border-radius: 10px;
+  height: calc(100%-10px);
```
Comment (claireolmstead) [inline frontend/src/features/BroadcastMedia/BroadcastMedia.module.css:9]:
```suggestion
```

File: frontend/src/features/BroadcastMedia/BroadcastMedia.tsx [hunk 3]
```diff
@@ -42,8 +42,9 @@ const BroadcastMedia = ({ attachments }: BroadcastMediaProps): ReactElement => {
               playsinline
               className={styles.image}
               url={tryUseIpfsGateway(attachment.url[0].href)}
-              width={670}
-              height={isVideo(attachment) ? 400 : 55}
+              width={'100%'}
```
Comment (claireolmstead) [inline frontend/src/features/BroadcastMedia/BroadcastMedia.tsx:45]:
```suggestion
```

File: frontend/src/features/BroadcastMedia/BroadcastMedia.tsx [hunk 4]
```diff
@@ -42,8 +42,8 @@ const BroadcastMedia = ({ attachments }: BroadcastMediaProps): ReactElement => {
               playsinline
               className={styles.image}
               url={tryUseIpfsGateway(attachment.url[0].href)}
-              width={670}
-              height={isVideo(attachment) ? 400 : 55}
+              maxWith={670}
```
Comment (mattheworris) [inline frontend/src/features/BroadcastMedia/BroadcastMedia.tsx:45]: typo?
```suggestion
              maxWidth={670}
```

**Review by mattheworris — APPROVED** [review]
- maybe a typo
🚢 it

**Pull request merged**